### ISSUE TITLE
Will it be available for Rails 4.1?

### ISSUE THREAD

**Issue by nasospsa** [apotonick/rspec-apotomo#18]
Specifically I haven't tested the compatibility but it seems to break on rspec-rails (needing 2.6 instead of 3.0.0)

Thanks in advance

**Comment by apotonick** [comment]
We just upgraded rspec-cells so maybe we can fix this, too. Feel free to help!!!! :)

**Comment by atambo** [comment]
https://github.com/apotonick/rspec-apotomo/pull/19 makes rspec-apotomo work with rspec3

**Comment by apotonick** [comment]
Are you using Cells 3 or Cells 4, @atambo ?

**Comment by atambo** [comment]
We are using cells 3 because apotomo's gemspec doesn't allow cells 4.

**Comment by apotonick** [comment]
Yeah, I know, because Apotomo uses the Cells 3 API, we have to rewrite/upgrade it, but I'm not sure it's worth it since I have zero idea who's using it... :grimacing:

**Comment by atambo** [comment]
Ya, unfortunately I'm stuck using it until I find the time to rewrite all our widgets using something else.

**Comment by apotonick** [comment]
...or we fix Apotomo...?

**Comment by atambo** [comment]
I'm not sure I really need apotomo. I feel like I could probably switch to just using cells. The only thing it's really missing is the ajax/url_for_event stuff.

**Comment by apotonick** [comment]
TBH, I feel like we should start working on `cells-widget`: a new gem that has the basic Apotomo features like event handlers and `url_for_event`, no/minimal nesting, done. Have been mulling about that for ages.

**Comment by atambo** [comment]
I would certainly switch to using/contributing to that :stuck_out_tongue:
